test/accel/IntegrationTestBase.cc

```diff
@@ -155,19 +166,27 @@ class EventAction final : public G4UserEventAction
 class ActionInitialization final : public G4VUserActionInitialization
 {
   public:
-    explicit ActionInitialization(IntegrationTestBase* test) : test_{test} {}
+    explicit ActionInitialization(IntegrationTestBase* test) : test_{test}
+    {
+        if (!CELER_USE_DEVICE && use_profiling())
```

Comment:
That smell fishy ... Is it actually semantically:
```suggestion
        if (use_profiling() && CELERITAS_ACTUALLY_USE_PERFETTO)
```
?
or can it just be
```suggestion
        if (use_profiling())
```

Reply:
Oh it should be perfetto, silly me...